app/Filament/Resources/UserResource/RelationManagers/FlightInformationRegionsRelationManager.php

```diff
@@ -45,7 +52,18 @@ public static function table(Table $table): Table
             ])
             ->headerActions([
                 Tables\Actions\CreateAction::make(),
-                Tables\Actions\AttachAction::make(),
+                Tables\Actions\AttachAction::make()
+                    ->label("Add User to FIR")
```

Comment:
Can this label be removed? I'm pretty sure we can live with `Attach` (default)

Reply:
Apologies, left it in when I was internally testing.